### PR TITLE
Back To The Future 1.0.0

### PR DESCRIPTION
**Pull request by ploch**

### CODE REVIEW

**Comment by francisdb** [comment]
Until now we have not been adding these changes to the patches.

**Comment by francisdb** [comment]
Is this for https://vpuniverse.com/files/file/15114-back-to-the-future-data-east-1990 ?

File: Back To The Future (Data East 1990) 1.0.0/Back To The Future (Data East 1990).vbs.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
++
+ '----- Phsyics Mods -----
+ Const RubberizerEnabled = 1			'0 = normal flip rubber, 1 = more lively rubber for flips
+ Const FlipperCoilRampupMode = 0   	'0 = fast, 1 = medium, 2 = slow (tap passes should work)
+\ No newline at end of file
+@@ -32,6 +44,7 @@
+ Const EnableMagnasave = 1		' 1 - on; 0 - off; if on then the magnasave button let's you rotate all LUT's
+ LoadLUT 'LUT
+ SetLUT 'LUT
++ImplicitDMD_Init 'DMD Enable
```
Comment (francisdb) [inline Back To The Future (Data East 1990) 1.0.0/Back To The Future (Data East 1990).vbs.patch:27]: this line is not needed

**Comment by jsm174** [comment]
Thanks for this!

At this time we are not adding new features to tables. This repo is just to show the minimum set of vbscript changes needed to get a table to run.

If this changes, I will re-open and merge.

**Pull request closed without merging**